tests/unittests/Foundation/BlockClassTests.mm

```diff
@@ -14,6 +14,8 @@
 //
 //******************************************************************************
 
+#ifdef _WIN32
```

Comment:
>_WIN32 [](start = 7, length = 6)

We might be able to get away with using `WINOBJC` for some of these, since that's what we use for client apps.